find.go

```diff
@@ -17,10 +18,11 @@ import (
 
 // DeviceQuery specifics a query for a network device.
 type DeviceQuery struct {
-	ModelName    string `json:"model_name"`
-	Manufacturer string `json:"manufacturer"`
-	SerialNumber string `json:"serial_number"`
-	Network      string `json:"network"`
+	ModelName    string   `json:"model_name"`
+	Manufacturer string   `json:"manufacturer"`
+	SerialNumber string   `json:"serial_number"`
+	Network      string   `json:"network"`
+	Endpoints    []string `json:"endpoints"`
```

Comment:
this isn't used

Reply:
Needed to keep these in the struct so a user can specify what endpoints they want in a discovery service. Intended to only be used in those, unless we find a way to actually discover them. otherwise I would need to wrap the queries and endpoints in another struct which could be more annoying to configure.

Reply:
ended up removing them. would need to test with hardware(no access to a upnp supported camera currently) to do one of the following 

1. validate endpoints a user set actually exist
2. discover ports/endpoints, assuming theres a consistent way to do this across devices.

Will still allow users to pre define endpoints in the discovery service